### PR TITLE
Update guidelines-for-gdpr-compliance.md

### PR DESCRIPTION
**Pull request by helen-laktionova**
Update from Anneliese Hartlaub

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/about/all/support/guidelines-for-gdpr-compliance.md [hunk 1]
```diff
@@ -11,32 +11,28 @@ redirect_from:
 
 {% info_block errorBox %}
 
-This document conforms to the [DSGVO](https://de.wikipedia.org/wiki/Datenschutz-Grundverordnung) and [RGPD](https://fr.wikipedia.org/wiki/R%C3%A8glement_g%C3%A9n%C3%A9ral_sur_la_protection_des_donn%C3%A9es) rules.
+This document conforms to the GDPR.
```
Comment (andriitserkovnyi) [inline docs/about/all/support/guidelines-for-gdpr-compliance.md:14]: Any particular reason we need to mention this?

File: docs/about/all/support/guidelines-for-gdpr-compliance.md [hunk 2]
```diff
@@ -11,32 +11,28 @@ redirect_from:
...
 
 {% endinfo_block %}
 
-As of May the 25th, 2018, the new General Data Protection Regulations (GDPR) take effect. This document describes how the Spryker Commerce OS is compliant with the GDPR.
+This document describes how the Spryker Services supports GDPR compliance of customers’ applications.
```
Comment (andriitserkovnyi) [inline docs/about/all/support/guidelines-for-gdpr-compliance.md:18]: Does `Spryker Services` refer to something specific?

File: docs/about/all/support/guidelines-for-gdpr-compliance.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Guidelines for GDPR compliance
```
Comment (andriitserkovnyi) [inline docs/about/all/support/guidelines-for-gdpr-compliance.md:2]: Who's the target audience of this doc?

Comment (helen-laktionova) [reply docs/about/all/support/guidelines-for-gdpr-compliance.md:2]: Answered in - https://spryker.slack.com/archives/C3KMWM1UZ/p1713176881720259?thread_ts=1712141501.987429&cid=C3KMWM1UZ

File: docs/about/all/support/guidelines-for-gdpr-compliance.md (hunk 1 above)
Comment (helen-laktionova) [reply docs/about/all/support/guidelines-for-gdpr-compliance.md:14]: Answered in - https://spryker.slack.com/archives/C3KMWM1UZ/p1713176881720259?thread_ts=1712141501.987429&cid=C3KMWM1UZ

File: docs/about/all/support/guidelines-for-gdpr-compliance.md (hunk 2 above)
Comment (helen-laktionova) [reply docs/about/all/support/guidelines-for-gdpr-compliance.md:18]: Asked Liz

**Pull request merged**